appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java

```diff
@@ -194,6 +196,11 @@ private void callDelegateWithOneError(
     byte[] result = null;
     final Double timeoutInSeconds = 10.0;
 
+    Logger logger = Logger.getLogger(VmApiProxyDelegate.class.getName());
+    Level level = logger.getLevel();
+    if (!logger.isLoggable(Level.FINE)) {
+      logger.setLevel(Level.SEVERE);
```

Comment:
Yes, I understand what it does but I do find it strange. E.g. what if it was INFO, is it OK to just leave it
as INFO at this case?